packages/sources/react/src/components/VtmnToggle/types.ts

```diff
@@ -0,0 +1 @@
+export type VtmnToggleSize = 'small' | 'medium';
```

Comment:
I would make an enumeration here no ?

Reply:
For others we did it like that with type.
I think using enum is more complicated for consuming it.
@lauthieb what's your think on it ?